framework_laptop.c

```diff
@@ -34,6 +34,7 @@
 
 static struct platform_device *fwdevice;
 static struct device *ec_device;
+static struct device *hwmon_dev;
```

Comment:
(I suspect I should do this for `ec_device` as well, but since that is a system resource that exists regardless of our platform device it felt like the wrong place at first.)